web-registry/src/components/organisms/BridgableEcocreditsTable/BridgableEcocreditsTable.styles.ts

```diff
@@ -0,0 +1,8 @@
+export const withLoaderStyles = {
+  display: 'flex',
+  width: '100%',
+  justifyContent: 'center',
+  flexDirection: 'col',
+  alignItems: 'center',
+  py: 25,
+};
```

Comment:
We can add `as SxProps<Theme>` to make this object typed.

Reply:
Great, thanks! I have also moved some styles to the styles directory, so they don't repeat and they are reusable